requirements.txt

```diff
@@ -1,5 +1,5 @@
 matplotlib
-numpy
+numpy==1.26.4
```

Comment:
Why pin numpy to this version number? Are numpys later than this non-functional with pyrk? If so, I recommend `numpy<=1.26.4`. If it needs to be at least that version fo numpy (and later versions are ok, but earlier ones aren't) then we would use the opposite (`numpy>=1.26.4`).

Reply:
I pinned it to 1.26.4 because errors kept popping up for a potential "binary incompatibility", which last I researched had to do with newer versions of numpy not working well with pandas. This currently seems to still be the case, at least for the test run in Python 3.9.